apps/backoffice-v2/package.json

```diff
@@ -119,13 +116,11 @@
     "i18next-http-backend": "^2.1.1",
     "jspdf": "^2.5.2",
     "jspdf-autotable": "^3.8.4",
-    "leaflet": "^1.9.4",
     "libphonenumber-js": "^1.10.49",
     "lodash-es": "^4.17.21",
     "lowlight": "^3.1.0",
     "lucide-react": "0.445.0",
     "match-sorter": "^6.3.1",
-    "msw": "^1.0.0",
```

Comment:
msw is a development only dependency. It's not supposed to affect build/deployment.

Reply:
It did

Reply:
i guess because you import it in the code..